packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentationTemplate/CustomerSegmentationTemplate.ts

```diff
@@ -30,8 +30,8 @@ type CustomerStandardMetafieldDependency = 'facts.birth_date';
 export interface CustomerSegmentationTemplateProps {
   /* Localized title of the template. */
   title: string;
-  /* Localized description of the template. */
-  description: string;
+  /* Localized description(s) of the template. */
+  description: string | string[];
```

Comment:
Should it just be a `string[]`? This way we don't have to check if you're getting a string or array in the component's implementation?

Reply:
sure! no strong opinion/i'm indifferent - @jonathanhamel4 brought up the advantage of a union type: it would mean it's backwards compatible. but i also think it is nice to keep it simple :)

Reply:
let's go with the union type (you both suggested it)